src/message.ts

```diff
@@ -72,12 +91,14 @@ export class MessageComputer {
         }
 
         const version = packedMessage.version || DEFAULT_MESSAGE_VERSION;
+        const signer = packedMessage.signer ?? SDK_JS_SIGNER;
```

Comment:
Here, for unpack, actually, we should not use the default if missing. Maybe, instead, `unknown`?

Reply:
CC: @ccorcoveanu

Reply:
That sounds good.